modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1,3 +1,27 @@
 = What’s New in Studio
 
 Anypoint Studio 7.15.0 improves the developer experience by enhancing Studio performance for a number of users, strengthens the asset import experience, and adds more to CloudHub deployment options.
```

Comment:
```suggestion
Anypoint Studio 7.15.0 improves the developer experience by enhancing Studio performance for Windows users, improves the import-asset experience, and adds more CloudHub deployment options.
```

Reply:
Is this sentence necessary?

Reply:
It works as a summary of the new features the new version of Studio has to avoid just listing features.